main.js

```diff
@@ -0,0 +1,354 @@
+        "headers": {
+            "cache-control": "no-cache",
+            "Postman-Token": "c8d081b1-fdbf-4fc7-836b-19fbead00753"
+        },
+        success: function( response ){
+            var data = Object.values(response.query.pages)[0].extract;
+            if (short_name === "Ale2" || short_name === "Stout2" || short_name === "Lager2" ||  short_name ==="Malt2" || short_name === "Non-Alcoholic Beer2") {
+                typesBeer[short_name] = data;
+
+            }
+            else{
+                renderingDescriptionOnDom(data, short_name);
+            }
+
+        }
+    };
+    $.ajax(ajaxConfig);
+}
+function callingStoreTypesOfBeer(){ ///////I HAVE STORED ALL 5 TYPES OF BEERS IN AN OBJECT SO WE CAN USE THEM FOR ERROR HANDLING
+    wikipediaApiSummary("ale beer", "Ale2");
+    wikipediaApiSummary("stout beer", "Stout2");
+    wikipediaApiSummary("lager beer", "Lager2");
+    wikipediaApiSummary("malt beer", "Malt2");
+    wikipediaApiSummary("Non-alcoholic drink", "Non-Alcoholic Beer2");
+}
+
+callingStoreTypesOfBeer(); ////GETTING THOSE BEERS IN THAT ARRAY FOR ERROR HANDLING
+
+function findingDescription(parameterRender1, parameterRender2) ///PARAM TAKES NAME OF BEER AND PASSES TO THE wikipediaApiSummary FUNCTION
+{
+    wikipediaApiSummary(parameterRender1, parameterRender2);
+}
+function renderingDescriptionOnDom(param1, param2){ ///PARAM1 IS THE DESCRIPTION(FROM WIKI) WE GOT FROM wikipediaApiSummary FUNCTION, PARAM2 IS ONE OF 5 BEERS, DEFAULT IF NO DESCRIPTION FROM WIKIPEDIA
+
+    if (!param1 || randomBeer.brewer === "Brick" || randomBeer.brewer === "Bavaria"){                       ///IF PARAM1 IS UNDEFINED (IT WOULD BE UNDEFINED IF THERE IS NO PAGE OR IF THE PAGE DOES NOT GET LOADED WHEN WE DO THE API CALL
+       param2 = `${param2}2`;               ///HAD TO DO THIS BECAUSE param2 ONLY EQUALS ex: ale AND WE NEED IT TO HAVE A 2 AT THE END TO USE IT AS A KEY VALUE PAIR FROM TYPES BEER OBJ
+        var errorRenderingFromTypesBeer = typesBeer[param2];
+        $('.encaseOfNoWiki').removeClass('displayingNone');
+       $('.encaseOfNoWiki').text(`${randomBeer.name} is a ${randomBeer.type} originally from ${randomBeer.country}. ${randomBeer.name} is brewed at ${randomBeer.brewer}. This beer has an alcohol percentage of ${randomBeer.abv}.`);
+        $('.wikipedia').html(`<span class="moreInfoWiki">More Information:</span> ${errorRenderingFromTypesBeer}`);
+   }
+
+   else{
+       $('.wikipedia').html(`<span class="moreInfoWiki">Brewery Information:</span> ${param1}`);
+        $('.encaseOfNoWiki').addClass('displayingNone');
+
+   }
+
+}
+
+
+function placesAPI(){
+    var theData = {
+        key: "AIzaSyAz5xq3SxTLX3I7l9jiA28_gfzQ05uB5ts",
+        input: randomBeer.brewer,
+        inputtype: "textquery",
+        fields: "photos,formatted_address,name,rating,opening_hours,geometry",
+    };
+    var placesAPIinput = {
+    dataType: "json",
+    url: "http://localhost:8888/c918_hackathon2/proxies/googleplaces.php",
```

Comment:
I won't fault you too much for this one but you dont need to path url like this. In fact it doesn't work like this on my computer anyways. All you need to do is point it to the proxy file itself, and assuming I have MAMP running, then it will work. Until you do get this hosted on a server you would want to write something about it your pull request about this. ex: "Please run mamp to get the full experience."
```suggestion
    url: "proxies/googleplaces.php",
```